eland/operations.py

```diff
@@ -340,7 +344,8 @@ def _metric_aggs(
         # Return single value if this is a series
         # if len(numeric_source_fields) == 1:
         #    return np.float64(results[numeric_source_fields[0]])
-        s = pd.Series(data=results, index=results.keys())
+        out_dtype = EMPTY_SERIES_DTYPE if not results else None
```

Comment:
Should probably create a helper function for creating a Series (I'll be using it more often with a local change I have).

That way it's tougher to forget to do this anywhere that series are being created.

Can use a signature of ...(data,  dtype=None, **kwargs) and replace dtype if data is empty